decred/decred/dcr/dcrdata.py

```diff
@@ -429,7 +429,7 @@ def __init__(self, db, netParams, datapath, skipConnect=False):
         self.heightMap = db.child("height", datatypes=("INTEGER", "BLOB"))
         self.headerDB = db.child("header", blobber=msgblock.BlockHeader)
         self.txBlockMap = db.child("blocklink")
-        self.tipHeight = None
+        self.tipHeight = -1
```

Comment:
This needs to be an integer and zero is a valid block id.

Reply:
Presumably the goal is a sentinel value indicating that the tip has not been retrieved yet. What you have is is fine, and I appreciate the effort to use the correct type, but checking for `self.tipHeight is None` is also nice syntax, so `None` is fine for a sentinel even for variables or properties in most cases, imo.

Reply:
Actually, scratch that. I see it's being used in comparisons without checking, so it has to be -1. My bad.